routes/levels.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const { addLevel, deleteLevel, getAllLevels } = require("../controllers/levels");
+const { validateLevelBody } = require("../middlewares/validators/levels");
+
+const router = express.Router();
+
+router.post("/", validateLevelBody, addLevel);
+router.delete("/:levelid", deleteLevel);
```

Comment:
we do not want everyone to create levels, authorize only superuser to do that

Reply:
done